docs/adding-a-managed-service.md

```diff
@@ -0,0 +1,568 @@
+---
+title: Adding a Managed Service
+toc: true
+weight: 460
```

Comment:
weight: 720
indent: true

so it's indented under the updated side-nav in #816 with "Developer Guide" at 710: https://github.com/crossplaneio/crossplane/blob/7ecc2df467b5c0683fe257a7cc1d316b76a9445a/docs/developer-guide.md